File-Formats-toc.html

```diff
@@ -249,7 +249,24 @@
 							</li>
 							<li>
 								<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-MAC">Mac</a>
-							</li>		
+							</li>
+							<li>
+								<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-Azure">Azure</a>
+								<ul>
+									<li>
+										<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-Functions-v1">Azure Function v1</a>
```

Comment:
It should be Azure Functions not Function

Reply:
Updated